Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json

```diff
@@ -3803,24 +3803,74 @@
 							"name": "GetAll_Assessment_ShouldUseNHibernateFilters",
 							"item": [
 								{
-									"name": "GetAll_Assessment_ShouldUseNHibernateFilters",
+									"name": "Get Assessments (default client)",
 									"event": [
 										{
 											"listen": "test",
 											"script": {
 												"exec": [
-													"    pm.test(\"Status code is 200\", () => {",
-													"    pm.expect(pm.response.code).to.equal(200);",
+													"pm.test(\"Status code is 200\", () => {",
+													"  pm.expect(pm.response.code).to.equal(200);",
 													"});",
 													"",
 													"pm.test(\"Should return some rows of assessment\", () => {",
-													"    const responseItems = pm.response.json();",
-													"    pm.expect(responseItems.length).to.be.greaterThan(0);",
+													"  const responseItems = pm.response.json();",
+													"  pm.expect(responseItems.length).to.be.greaterThan(0);",
+													"});",
+													""
+												],
+												"type": "text/javascript",
+												"packages": {}
+											}
+										},
+										{
+											"listen": "prerequest",
+											"script": {
+												"exec": [
+													""
+												],
+												"type": "text/javascript",
+												"packages": {}
+											}
+										}
+									],
+									"request": {
+										"method": "GET",
+										"header": [],
+										"url": {
+											"raw": "{{ApiBaseUrl}}/data/v3/ed-fi/assessments",
+											"host": [
+												"{{ApiBaseUrl}}"
+											],
+											"path": [
+												"data",
+												"v3",
+												"ed-fi",
+												"assessments"
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "Get Assessments (other namespace client)",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"exec": [
+													"pm.test(\"Status code is 200\", () => {",
+													"  pm.expect(pm.response.code).to.equal(200);",
 													"});",
 													"",
+													"pm.test(\"Should return no items\", () => {",
+													"  const responseItems = pm.response.json();",
+													"  pm.expect(responseItems.length).to.equals(0);",
```

Comment:
This test was added to demonstrate that the namespace filtering is working (contrasted with [the line above](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/pull/1153/files#diff-aa12a53e29dc6d1da68ea8eb695272d4795f44263656802f9bd5a9ef6bc95b0bR3818) where greater than 0 items are returned.